EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigiGPU.cc

```diff
@@ -127,16 +128,26 @@ SiPixelRawToDigiGPU::SiPixelRawToDigiGPU( const edm::ParameterSet& conf )
   cudaMallocHost(&pdigi_h,    sizeof(uint32_t)*WSIZE);
   cudaMallocHost(&rawIdArr_h, sizeof(uint32_t)*WSIZE);
 
-  cudaMallocHost(&errType_h,  sizeof(uint32_t)*WSIZE);
-  cudaMallocHost(&errRawID_h, sizeof(uint32_t)*WSIZE);
-  cudaMallocHost(&errWord_h,  sizeof(uint32_t)*WSIZE);
-  cudaMallocHost(&errFedID_h, sizeof(uint32_t)*WSIZE);
+  uint32_t VSIZE = sizeof(GPU::SimpleVector<error_obj>);
+  uint32_t ESIZE = sizeof(error_obj);
+  bool success = cudaMallocHost(&error_h, VSIZE) == cudaSuccess &&
```

Comment:
can you add calls to `cudaCheck()` around all `cudaMallocHost()`, and remove the `assert` ?